README.md

```diff
@@ -127,6 +127,10 @@ Recently we have been actively improving our query engine primarily for better c
 
 Please refer to the [SQL Language Reference Manual](./docs/user/index.rst), [Piped Processing Language (PPL) Reference Manual](./docs/user/ppl/index.rst) and [Technical Documentation](https://opensearch.org/docs/latest/search-plugins/sql/index/) for detailed information on installing and configuring plugin.
 
+## Forum
```

Comment:
Thanks for the update!  I like this a lot better.  

Would you mind adding a link in the table of contents at the top of the page (lines 6-7)?

Reply:
Done!